parlai/crowdsourcing/utils/tests.py

```diff
@@ -133,22 +133,30 @@ def _register_mock_agents(self, num_agents: int = 1) -> List[str]:
 
         for idx in range(num_agents):
 
-            # Register the worker
             mock_worker_name = f"MOCK_WORKER_{idx:d}"
             max_num_tries = 6
             initial_wait_time = 0.5  # In seconds
             num_tries = 0
             wait_time = initial_wait_time
             while num_tries < max_num_tries:
                 try:
+
```

Comment:
Unrelated, but you can get rid of mypy errors in this file by providing typing for self, like:
```python
def _check_output_key(self: Union[<testcase class you're mixing in with>, 'AbstractParlAIChatTest'], key: str, actual_value: Any, expected_value: Any):
```

Reply:
Good point - changed